manimlib/mobject/matrix.py

```diff
@@ -100,8 +102,8 @@ def organize_mob_matrix(self, matrix):
                 )
         return self
 
-    def add_brackets(self):
-        bracket_pair = TexMobject("\\big[", "\\big]")
+    def add_brackets(self, left, right):
```

Comment:
Should probably default them to `r"\big[", r"\big]"`, jic (also would make this a non-breaking change)

Reply:
Agreed

Reply:
Fine with my. Sorry for asking, but it seems to me that your suggested change does not use `r"\big["` and `r"\big]"` contrary to what you wrote in your comment. What do we do?

Reply:
That's because he used raw strings in his comment but non-raw strings in the edit; they do the same thing.

Reply:
@safinsingh Thanks, so I misunderstood. I thought his suggestion was mainly about using raw strings :)

Reply:
yeah sorry 
I prefer raw strings but I wanted to be consistent with the already-existing code